src/OpenTelemetry.Shims.OpenTracing/ScopeManagerShim.cs

```diff
@@ -46,18 +49,18 @@ public ScopeManagerShim(Trace.Tracer tracer)
         {
             get
             {
-                var currentSpan = this.tracer.CurrentSpan;
-                if (currentSpan == null || !currentSpan.Context.IsValid)
+                var currentActivity = Activity.Current;
```

Comment:
TracerNew.Current instead of Activity.Current.

Reply:
This is not possible. In all the `constructor`, I take span related details and changed to activity. Whole implementation is working with Activity.